src/components/withTooltip.js

```diff
@@ -1,23 +1,57 @@
-import React, { Component, Fragment } from 'react';
+import React, { Component } from 'react';
 import { ListTooltip } from 'ot-ui';
+import { theme } from 'ot-charts';
```

Comment:
It would be better not to have to use `theme` from `ot-charts` here. This could perhaps be solved by changing the interface of `withTooltip` and moving it into `ot-charts`. Usage would be something like:
```
const Tooltip = ({ data, open, anchorEl }) => {
  const dataList = ... // dependence on tableColumns
  return <ListTooltip open={open} anchorEl={anchorEl} dataList={dataList} />;
}
const manhattanWithTooltip = withTooltip(Manhattan, Tooltip)
```

In the above, `Tooltip` should not need to know what highlighting to change. That should be handled by the chart `Manhattan` ideally.